scripts/conda-requirements.yaml

```
@@ -0,0 +1,57 @@
+name: chipyard
+channels:
+  - conda-forge
+  - ucb-bar
```

```suggestion
  - ucb-bar
  - nodefaults
```

I think to avoid any confusion about where you've installed conda from (Anaconda Inc vs miniforge installers), we should explicitly add the nodefaults meta-channel that prevents usage of the 'defaults' channel as documented at https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-file-manually